docker-compose.yaml

```diff
@@ -0,0 +1,44 @@
+version: "3.8"
+
+services:
+  app:
+    container_name: frontend-dev
+    build:
+      context: .
+      target: development
+    volumes:
+      - ./app:/app
+    env_file: ./.env
+    ports:
+      - ${NODE_LOCAL_PORT}:${NODE_DOCKER_PORT}
+    networks:
+      - frontend
+      - backend
+
+  #  flask:
+  #    build:
+  #      context: backend
+  #      target: builder
+  #    restart: always
+  #    env_file: ./.env
+  #    networks:
+  #      - backend
+  #    ports:
+  #      - ${FLASK_LOCAL_PORT}:${FLASK_DOCKER_PORT}
+
+  nginx:
+    container_name: nginx-server
+    image: nginx:latest
```

Comment:
I would pin this at an LTS version rather than `latest`.

It requires us to keep things up-to-date, but ensures that we don't accidentally get breaking changes.